index.html

```diff
@@ -516,16 +516,19 @@
           Non-normative note: Variability among client types and locations may mean that LDP-NR content is addressed in
           ways that are external to the Fedora server but not resolvable by all clients. This specification describes
           the use of <code>Content-Type: message/external-body</code> values to signal, on POST or PUT, that the Fedora
-          server should should not consider the request entity to be the LDP-NR's content, but that a
+          server should not consider the request entity to be the LDP-NR's content, but that a
           <code>Content-Type</code> value will signal a name or address at which the content might be retrieved. The
           <code>url</code> and <code>local-file</code> <code>access-type</code> values motivate this specification, but
           a Fedora server may support any <code>access-type</code> parameters per the requirements for advertisement
           and rejection specified here.
         </blockquote>
         <p id='message-external-body-accept-post'>
-          Fedora servers that support LDP-NR with <code>message/external-body</code> MUST advertise that support in the
-          <code>Accept-Post</code> response header for each supported <code>access-type</code> parameter of supported
-          <code>Content-Type</code> values.
+          Fedora servers MUST support the creation of LDP-NRs with <code>Content-Type</code> of
```

Comment:
I agree with all of the changes here except the addition of this MUST. The other changes are good, even necessary, clarification about what must be done of external content is supported, but the use case is not universal and its support is advertise-able.

Reply:
@fcrepo/editors : this specific topic of tightening the language around external content dates back to:
https://wiki.duraspace.org/display/FF/2017-09-27+-+API+Spec+Meeting

The conversations in meetings since 9/27 have included requiring the support for external content. I would like to get @fcrepo/editors input on this suggestion of relaxing that requirement before taking action.

Reply:
@fcrepo/editors : Looking for input.

Reply:
I don't have a strong opinion on this point.  I would be fine with relaxing that to a SHOULD, for instance.

Reply:
I'm still quite conflicted about what we set as a minimum bar to be "Fedora". In the abstract I agree completely with @barmintor that a client or test-suite can tell whether this feature is supported (I'm imaging a test suite that outputs a feature implementation checklist...) so in that sense SHOULD makes perfect sense. A community agreed set of must-meet vs. optional use cases would really be the right way to arbitrate this but since we don't have that perhaps it is best to go with optional, relax to SHOULD.

Reply:
I completely agree with @zimeon here — it would be great to have more solid info about external content.  I know many Samvera repos use it.  But I'm not sure if we need to go so far as to say that all Fedora implementations must support it.

Related to this, I thought we'd coalesced around requiring external content be proxied.  But the discussion in #237 includes discussion of redirecting as an alternative.  The main reason I bring this up is that redirecting seems like a lower bar than proxying, so it might make the difference for me.

Reply:
I believe the consensus is that the Fedora impls `MUST` support external content, but only `SHOULD` support API creation of such content.
I am happy to change `MUST` to `SHOULD` on [line:526](https://github.com/fcrepo/fcrepo-specification/pull/242/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR526).

Reply:
@barmintor : I believe your requested change has come true.